src/Workspaces/Core/Desktop/TypeForwarders.cs

```diff
@@ -0,0 +1,8 @@
+﻿#if false
```

Comment:
:memo: Two things:

1. File header validation will expect the file header to precede the preprocessor directive
2. Why `false` / why include the file if it's fully excluded

Reply:
This is only temporary. The PR is blocked until the Roslyn analyzers are updated to bring in a newer public API analyzer that supports type forwarders. So, this won't be merged like this.

Reply:
💭 I thought we fixed that last year in dotnet/roslyn-analyzers#1132

Reply:
We did. The analyzers used by the roslyn repo have not been updated since before then. 😞 

Waiting on this: https://github.com/dotnet/roslyn/pull/25179

Reply:
@mavasani is trying another roslyn analyzers update with reduced scope here: https://github.com/dotnet/roslyn/pull/25891. Hopefully that will be able to go in soonish and unblock the type forwarder.

Reply:
It turns out there was a bug in the type forwarder support in the public API analyzer for properties. This PR will include a workaround for that issue and I've submitted https://github.com/dotnet/roslyn-analyzers/pull/1657 with a fix.